docs/redirects.yaml

```diff
@@ -0,0 +1,35 @@
+"en/latest/":
```

Comment:
@dwgebler's recent change in https://github.com/ClusterHQ/flocker/pull/2503 makes `/en/latest` stop being a redirect, and instead be the canonical location for the docs, fwiw.